src/parser.ts

```diff
@@ -52,7 +52,7 @@ export default class Parser {
                     node.selector
                         .split(/[\s,~+>]+/)
                         .forEach((str) => {
-                            const strs = str.match(/[.#][^.#]+/g);
+                            const strs = str.match(/[.#][^.#|)]+/g);
```

Comment:
"pipe" symbol is not required here.

Reply:
Possible changing parent regexp from split method takes some result.
```js
.split(/[\s,~+>]+/)
```
=>
```js
.split(/[\s,~+>()]+/)
```
I guess this place is better.